.github/workflows/ci.yml

```diff
@@ -54,11 +62,71 @@ jobs:
       - name: check copyright headers
         run: bash .github/copyright.sh
 
+  clippy-stable:
+    name: cargo clippy
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, macos-14, ubuntu-latest]
```

Comment:
```suggestion
        # TODO: Switch to macos-latest after macos-14 is latest.
        os: [windows-latest, macos-14, ubuntu-latest]
```

Reply:
There are three places that it needs to be changed right now, and I expect that count to grow soon with [rfcs#5](https://github.com/linebender/rfcs/pull/5). That's why I added the note about this to the top of the file, as opposed to one of those numerous places.